code/modules/supermatter/supermatter.dm

```diff
@@ -71,6 +71,12 @@
 	var/list/message_exclusions = list(/obj/effect/effect/sparks,/obj/effect/overlay/hologram)
 	machine_flags = MULTITOOL_MENU
 
+/obj/machinery/power/supermatter/airflow_hit(atom/A)
+	if(ismovable(A))
+		var/atom/movable/movingA = A
```

Comment:
do you really need to typecast it

Reply:
I think I need the ismovable because Bumped will only take atom/movables.

Reply:
I could just go straight to consume, rather than Bumped, but then there will be some REALLY "fun" interactions, such as I believe it being able to just immovable rod the station.

Reply:
ill just remove the check.

Reply:
so, lets keep the check for now.

Reply:
works for me

Reply:
i didn't mean the check, just the typecasting but still, whatever.